frontend/src/app/components/crm/leads/leads.component.ts

```diff
@@ -164,6 +168,11 @@ export class LeadsComponent implements AfterViewInit {
       });
   }
 
+  sendEmail() {
+              window.open(
+                "https://mail.google.com/mail/u/0/?view=cm&fs=1&tf=1&source=mailto&su=Greetings", "_blank");
+  }
+
```

Comment:
In the future, make dynamic (Send emails to the correct lead(s))

Reply:
Yes, I'm working on that.